hadoop-hdfs-project/hadoop-hdfs-httpfs/src/main/java/org/apache/hadoop/fs/http/server/HttpFSExceptionProvider.java

```diff
@@ -95,4 +99,9 @@ protected void log(Response.Status status, Throwable throwable) {
     LOG.warn("[{}:{}] response [{}] {}", method, path, status, message, throwable);
   }
 
+  private void logErrorFully(Response.Status status, Throwable throwable) {
+    if (LOG.isDebugEnabled()) {
+      LOG.debug("Failed with " + status, throwable);
```

Comment:
Let's use parameterized logging, shall we?
That way, you don't need to check if debug log is enabled.

Reply:
Thanks @jojochuang 
I pushed a new commit with the fix.